ibis/file/parquet.py

```diff
@@ -0,0 +1,113 @@
+import ibis
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+from ibis.file.client import FileClient
+from ibis.pandas.core import pre_execute, execute
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+
+def connect(dictionary):
+        return ParquetClient(dictionary)
+
+
+_ARROW_DTYPE_TO_IBIS_TYPE = {
+    'int8': dt.int8,
```

Comment:
Can we use the arrow objects here rather than strings?

Reply:
This seems not implemented :>
```
In [5]: hash(pa.int8())
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-bca0e6e2f6af> in <module>()
----> 1 hash(pa.int8())

TypeError: unhashable type: 'pyarrow.lib.DataType'
```